metaissue.go

```diff
@@ -48,6 +48,18 @@ type MetaIssueType struct {
 	Fields      tcontainer.MarshalMap `json:"fields,omitempty"`
 }
 
+type MetaIssueTypeDetails struct {
+	IsLast     bool                    `json:"isLast,omitempty" yaml:"isLast,omitempty"`
+	MaxResults int                     `json:"maxResults,omitempty" yaml:"maxResults,omitempty"`
+	StartAt    int                     `json:"startAt,omitempty" yaml:"startAt,omitempty"`
+	Total      int                     `json:"total,omitempty" yaml:"total,omitempty"`
+	Values     []tcontainer.MarshalMap `json:"values,omitempty"`
```

Comment:
missing yaml tag? or does it not need one?

Reply:
nice catch! we don't need yaml here for Jira API structs.